pom.xml

```diff
@@ -741,6 +742,12 @@
       <artifactId>simple-java-mail</artifactId>
       <version>5.0.0</version>
     </dependency>
+    <dependency>
+      <groupId>net.markenwerk</groupId>
+      <artifactId>utils-mail-dkim</artifactId>
+      <version>1.1.12</version>
```

Comment:
The versions should be in properties at the top sorted alphabetically and into non-test/test. Same for above for `simple-java-mail` and below for `dumbster`.

Reply:
Why is this not a `test` scope dependency?